src/config.rs

```diff
@@ -0,0 +1,83 @@
+use crate::args::Args;
+use crate::errors::*;
+use serde::Deserialize;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+const DEFAULT_URL: &str = "https://security.archlinux.org/all.json";
+
+pub struct Config {
+    pub source: String,
+    pub proxy: Option<String>,
+}
+
+impl Config {
+    pub fn load(args: &Args) -> Result<Self> {
+        let mut merged = ConfigFile::default();
+
+        let mut configs = vec![PathBuf::from("/etc/arch-audit.conf")];
+        if let Some(dir) = dirs_next::config_dir() {
+            let path = dir.join("arch-audit.conf");
```

Comment:
I would prefer to namespace the configs into an `arch-audit` directory for both locations. maybe the file should also be `settings.conf` or `settings.toml` or similar, whose mime-type would work best?